### DISCUSSION TITLE
making NFC detections clickable

### DISCUSSION THREAD

**Discussion by YvedD** [Nachtzuster/BirdNET-Pi#127] [Ideas]
Hi all !
First things first : awesome fork of the original Birdnet-Pi !
So many thanks to all who put time and effort in this, thx !

I have a question (idea if you want):

Since 2 months i'm using the birdnet-pi on a RPi 4 4Gb to track any Nocturnal Flight Calls (NFC's) but still on the original fork (will migrate this weekend as we speak).

It's a great hobby and i really love it.

I was thinking if it would be possible to click the detections directly in the daily overview page, so that one can get immediatly to the recordings of a species in a specific time window to get to the recordings?
That way one could get really fast to the according soundclips for examination. Might be a great thing in full migration periods to improve the workflows and write all detections down in "trektellen.org"

Grtz,
Yves

![Screenshot 2024-06-29 19 28 41](https://github.com/Nachtzuster/BirdNET-Pi/assets/24252175/0934031a-67dd-4cbe-a77c-ca03f41af930)

Edit : Migrating to this fork went very well, great job !!

**Comment by alexbelgium** [comment]
Hi, to do that usually I go in the Today's Detections and click on those specific species. Currently it is an image, so it is not clickable unless a very significant rewrite of this part to make it a dynamic non-png based chart.

However, you can set apprise to send you a notification for each of those species ; and then you receive a link that when clicked sends you directly to the observation itself to review it.

**Comment by Nachtzuster** [comment]
> Edit : Migrating to this fork went very well, great job !!

:heart:

**Comment by Nachtzuster** [comment]
That would be a great feature, indeed.
For now the search bar might help you out a bit:
![image](https://github.com/Nachtzuster/BirdNET-Pi/assets/5863099/3f4eb24f-7ec1-4aac-9ad0-b0f6ef0f9a97)

Note the dash to avoid false matches on minutes and seconds

**Comment by lloydbayley** [comment]
I like the idea very-much, but not-so-much the coding to facilitate it. As @alexbelgium says, it would be major. In the past, I've seen co-ordinates on images made clickable but it's unbelievably hit and miss and I'm not even sure if it's still do-able without some serious additions. Would you be upset if we put it in the "to think about" section and you use @Nachtzuster s suggestion for now?

I must also confess that, for a fraction of a second, when I read NFC, I thought, my god, now we have to start electronically tagging birds by detecting them and firing NFC tags at them as the fly over! How are we going to do the targeting? Surely, this is beyond the BirdNet-Pi scope! :)

**Reply by alexbelgium** [reply]
Ahah NFC tagging of birds could be fun - this is not only a blackbird, this is Mike the blackbird!

Actually is there a way to make a suggestion/ideas list in github? I had other ideas to list without really wanting to work on ;)

**Reply by lloydbayley** [reply]
I'm not allowing it unless you get to make the PEW PEW noise when you shoot the NFC tags at them! :)
As for the ideas, I think it's limited to the inbuilt issue-types...Enhancements etc

**Reply by YvedD** [reply]
> I like the idea very-much, but not-so-much the coding to facilitate it. As @alexbelgium says, it would be major. In the past, I've seen co-ordinates on images made clickable but it's unbelievably hit and miss and I'm not even sure if it's still do-able without some serious additions. Would you be upset if we put it in the "to think about" section and you use @Nachtzuster s suggestion for now?
> 
> I must also confess that, for a fraction of a second, when I read NFC, I thought, my god, now we have to start electronically tagging birds by detecting them and firing NFC tags at them as the fly over! How are we going to do the targeting? Surely, this is beyond the BirdNet-Pi scope! :)

Would a separate menubar item be a option "check NFC" where a query on the table would look for nighttime recordings only?
Maybe there is already some sort of script or sql command somewhere that dynamiccaly checks for nightly hours?

**Comment by DougWelch** [comment]
https://github.com/Mattk70/Chirpity-Electron is a very useful tool for 
NFC detections and already has this functionality built-in.

It is available for Windows, MacOS (Apple Silicon only), and can be run 
in Linux if you follow the Development notes.

Cheers,
Doug

On 7/1/24 04:58, Alexandre wrote:
>
> 	
> Caution: External email.
>
>
> Ahah NFC tagging of birds could be fun - this is not only a blackbird, 
> this is Mike the blackbird!
>
> Actually is there a way to make a suggestion/ideas list in github? I 
> had other ideas to list without really wanting to work on ;)
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/Nachtzuster/BirdNET-Pi/discussions/127#discussioncomment-9923297>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAYGKVVWM7VDHRVG3X5D35DZKEKZVAVCNFSM6AAAAABKDLS45CVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMRTGI4TO>.
> You are receiving this because you are subscribed to this 
> thread.Message ID: 
> ***@***.***>
>

--------------HXBysO7Bh5OSMaT9MqgJvbYh
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: 8bit

<!DOCTYPE html><html><head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  </head>
  <body>
    <br>
    <a class="moz-txt-link-freetext" href="https://github.com/Mattk70/Chirpity-Electron">https://github.com/Mattk70/Chirpity-Electron</a> is a very useful tool
    for NFC detections and already has this functionality built-in.<br>
    <br>
    It is available for Windows, MacOS (Apple Silicon only), and can be
    run in Linux if you follow the Development notes.<br>
    <br>
    Cheers,<br>
    Doug<br>
    <br>
    <div class="moz-cite-prefix">On 7/1/24 04:58, Alexandre wrote:<br>
    </div>
    <blockquote type="cite" ***@***.***">
      
      
      <table width="100%" cellspacing="0" cellpadding="0" border="0" align="left">
        <tbody>
          <tr>
            <td style="background:#7A003C;padding:5pt 2pt 5pt 2pt"><br>
            </td>
            <td cellpadding="7px 6px 7px 15px" style="background:#ff000;padding:5pt 4pt 5pt 12pt;word-wrap:break-word" width="100%">
              <div style="color:#000000;"><span style="color:#000000; font-weight:bold;">Caution:
                </span>External email. </div>
            </td>
          </tr>
        </tbody>
      </table>
      <br>
      <div>
        <p dir="auto">Ahah NFC tagging of birds could be fun - this is
          not only a blackbird, this is Mike the blackbird!</p>
        <p dir="auto">Actually is there a way to make a suggestion/ideas
          list in github? I had other ideas to list without really
          wanting to work on ;)</p>
        <p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">—<br>
          Reply to this email directly, <a href="https://github.com/Nachtzuster/BirdNET-Pi/discussions/127#discussioncomment-9923297" moz-do-not-send="true">
            view it on GitHub</a>, or <a href="https://github.com/notifications/unsubscribe-auth/AAYGKVVWM7VDHRVG3X5D35DZKEKZVAVCNFSM6AAAAABKDLS45CVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMRTGI4TO" moz-do-not-send="true">
            unsubscribe</a>.<br>
          You are receiving this because you are subscribed to this
          thread.<img src="https://github.com/notifications/beacon/AAYGKVTONRJKHRLBHHABDZLZKEKZVA5CNFSM6AAAAABKDLS45CWGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQAS5VOC.gif" alt="" moz-do-not-send="true" width="1" height="1"><span style="color: transparent; font-size: 0; display: none; visibility: hidden; overflow: hidden; opacity: 0; width: 0; height: 0; max-width: 0; max-height: 0; mso-hide: all">Message
            ID: <span>&lt;Nachtzuster/BirdNET-Pi/repo-discussions/127/comments/9923297</span><span>@</span><span>github</span><span>.</span><span>com&gt;</span></span></p>
        <script type="application/ld+json">[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/Nachtzuster/BirdNET-Pi/discussions/127#discussioncomment-9923297",
"url": "https://github.com/Nachtzuster/BirdNET-Pi/discussions/127#discussioncomment-9923297",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]</script></div>
    </blockquote>
    <br>
  </body>
</html>

--------------HXBysO7Bh5OSMaT9MqgJvbYh--

**Reply by YvedD** [reply]
I know, i already use it...after i downloaded the files from birdnet-pi.
But i get to much false positives a.t.m.
It is also not relevant to my first question about filtering out NFC´s only.

Grtz and thanks anyway
Yves
